core/domain/state_domain_test.py

```diff
@@ -486,7 +493,9 @@ def test_solution_validation(self):
         # Solution should be set to None as default.
         self.assertEqual(init_state.interaction.solution, None)
 
-        init_state.add_hint(state_domain.SubtitledHtml('hint_1', {}))
+        init_state.interaction.hints.append(
```

Comment:
Ditto above.

Reply:
are we expecting to have "update_interaction_hints" over here?

Reply:
@DubeySandeep I am extremely sorry for missing out on this. Will not happen again.
Thanks.